website/docs/reference/source-properties.md

```diff
@@ -8,9 +8,13 @@ description: "Learn how to use source properties in dbt."
 - [Declaring resource properties](/reference/configs-and-properties)
 
 ## Overview
-Source properties can be declared in `.yml` files in your `models/` directory (as defined by the [`model-paths` config](/reference/project-configs/model-paths)).
 
-You can name these files `whatever_you_want.yml`, and nest them arbitrarily deeply in subfolders within the `models/` directory.
+import PropsCallout from '/snippets/_config-prop-callout.md';
```

Comment:
i'm pretty sure source properties shouldn't have this 'not supported in `dbt_project.yml` callout but [this doc](https://docs.getdbt.com/reference/configs-and-properties#which-properties-are-not-also-configs) made it seem like it didn't support it - did i misunderstand the doc?

Reply:
oh ok gotcha1